src/rust/receiver-mock/src/router.rs

```diff
@@ -167,6 +167,82 @@ pub async fn handler_terraform_fields_quota() -> impl Responder {
     })
 }
 
+pub async fn handler_terraform_fields() -> impl Responder {
+    #![allow(non_snake_case)]
```

Comment:
why?

Reply:
Do want to achieve this?
https://serde.rs/attr-rename.html

Reply:
![image](https://user-images.githubusercontent.com/69143962/99796917-97311b80-2b2e-11eb-8651-1d7f935c6f75.png)